docs/mp/grpc/server-side-services.adoc

```diff
@@ -260,23 +284,21 @@ public class StringService
     extends StringServiceGrpc.StringServiceImplBase {
 ----
 
-In exactly the same way, if a class is an implementation of `io.helidon.grpc.server.GrpcService` then by annotating the class with
-the `@ApplicationScoped` annotation it will be discovered and deployed when the MP gRPC server starts.
+In exactly the same way, if a class is an implementation of `io.helidon.grpc.server.GrpcService`, then it will be discovered and deployed when the MP gRPC server starts by simply annotating the class with the `@ApplicationScoped` annotation.
 
 [source,java]
 ----
 @ApplicationScoped
 public class StringService implements GrpcService {
 ----
 
-=== Implement a GrpcMpExtension
-If it is not possible to annotate the service class (for example the code is built by a third party) another way to deploy none
+==== Implement a GrpcMpExtension
+If it is not possible to annotate the service class (for example the code is built by a third party), another way to deploy none
```

Comment:
do you mean **non** CDI beans?

another way to deploy none CDI bean services is to...

Reply:
Should be corrected in PR: https://github.com/oracle/helidon/pull/4606